source/nodejs/adaptivecards/src/card-object.ts

```diff
@@ -136,6 +145,19 @@ export abstract class CardObject extends SerializableObject {
         return result;
     }
 
+    /**
+     * Finds the innermost CardObject who's renderedElement is equal to or parent of the query Node.
```

Comment:
Assuming the method name is changed to `findDomNodeOwner` as I suggested:

- Finds the innermost CardObject that owns the specified DOM node.
- @param node The DOM node to look for.
- @returns The CardObject owner of the specified DOM node, or undefined if no owner was found.